package.json

```diff
@@ -84,7 +84,8 @@
     "#": "needed `any`s",
     "ignoreFiles": [
       "lib/index.d.ts",
-      "lib/index.js"
+      "lib/index.js",
+      "index.d.ts"
```

Comment:
Why did this change?

Reply:
because of the `Processor<any>` export: https://github.com/rehypejs/rehype-remark/blob/3192063f5dd0fdbd5214f4813735762a61a618c6/lib/index.js#L5.
Probably not super useful, but as `Options` are exported, `Processor` makes sense as well.